.eslintrc.cjs

```diff
@@ -8,7 +8,7 @@ module.exports = {
     node: true,
   },
   ignorePatterns: [
-    "packages/*/dist/**",
+    "packages/**/dist/**",
```

Comment:
Hm, I'd _really_ like to avoid this. It breaks with the monorepo pattern, and scripts/set-workspace-version.js won't update our dependencies.

Can we simplify the structure somehow?